frontend/app/controllers/Joiner.scala

```diff
@@ -272,6 +282,7 @@ object Joiner extends Controller with ActivityTracking
   }
 
   def thankyou(tier: Tier, upgrade: Boolean = false) = SubscriptionAction.async { implicit request =>
+    implicit val resolution: TouchpointBackend.Resolution = TouchpointBackend.forRequest(PreSigninTestCookie, request.cookies)
```

Comment:
Fair enough - there will eventually be 3 endpoints that use this method, is it ok if I look for commonality or a refactor when they're all in place?